lib/conversion.ts

```diff
@@ -1,6 +1,9 @@
-import { InsightsEvent } from "./_sendEvent";
+export interface InsightsSearchConversionEvent {
+  eventName: string;
+  userID: string;
+  timestamp: number;
+  indexName: string;
 
-export interface InsightsSearchConversionEvent extends InsightsEvent {
   objectID: string | number;
```

Comment:
We cannot provide multiple `objectID`?

Reply:
I thought I would do that later but I can add it here to this PR